bundles/org.openhab.binding.energidataservice/src/main/java/org/openhab/binding/energidataservice/internal/ApiController.java

```diff
@@ -110,7 +111,7 @@ public ElspotpriceRecord[] getSpotPrices(String priceArea, Currency currency, Da
         }
 
         Request request = httpClient.newRequest(ENDPOINT + DATASET_PATH + DATASET_NAME_SPOT_PRICES)
-                .param("start", start.toString()) //
+                .timeout(REQUEST_TIMEOUT_MS, TimeUnit.MILLISECONDS).param("start", start.toString()) //
```

Comment:
Formatting and unit commented above.
```suggestion
                .timeout(REQUEST_TIMEOUT_SECONDS, TimeUnit.SECONDS) //
                .param("start", start.toString()) //
```

Reply:
Done